.github/workflows/tests.yml

```diff
@@ -6,15 +6,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: [3.7, 3.8, 3.9, "3.10"]
-        plone: ["6.0.0b3", "5.2.9"]
+        python: [3.7, 3.8, 3.9, "3.10", "3.11"]
```

Comment:
@davisagli I guess we should reflect this in the changelog as well as in setup.py, what do you think?

Reply:
@davisagli ignore my comment. Seems that @mauritsvanrees added the classifiers for Python 3.10 and 3.11 some time ago already. We just did not test 3.11 properly so far. Thanks for fixing this!